packages/@vivliostyle/theme-base/css/common/meta-properties.css

```
@@ -0,0 +1,56 @@
+:root {
+  /**
+   * Typography
+   */
+  --vs-font-family: sans-serif;
+  --vs-font-feature-settings: 'kern', 'liga', 'calt';
+  --vs-font-size: 16px;
+  --vs-font-size-smaller: calc(var(--vs-font-size) * 0.8);
+  --vs-font-style: normal;
+  --vs-font-weight-normal: 400;
+  --vs-font-weight-bold: 700;
+  --vs-hanging-punctuation: allow-end;
+  --vs-hyphens: auto;
+  --vs-line-break: strict;
+  --vs-line-height: 1.5;
+  --vs-letter-spacing: 0;
+  --vs-overflow-wrap: break-word;
+  --vs-tab-size: 4;
+  --vs-text-align: auto;
```

`auto` は `text-align` プロパティの正しい値でありません。

```suggestion
  --vs-text-align: justify;
```

あるいはルートでの指定は `start` として、本文の段落など必要箇所で `justify` とする。

---

デフォルトのtext-alignを `start` に変更しました。個人的には文章全体でjustifyされる動作は違和感があったので、ルートでの指定は `start` として、pタグに対して `justify` を適用するようにしました。

https://github.com/vivliostyle/themes/blob/c70e5fe79f972b34a17d390a22107052ae69eeda/packages/%40vivliostyle/theme-base/css/common/meta-properties.css#L25
https://github.com/vivliostyle/themes/blob/c70e5fe79f972b34a17d390a22107052ae69eeda/packages/%40vivliostyle/theme-base/css/common/basic.css#L58